pulp-api/Dockerfile

```diff
@@ -4,5 +4,9 @@ FROM ${FROM_ORG}pulp-core:$FROM_TAG
 
 ADD entrypoint.sh /entrypoint.sh
 
+ADD signing.py /signing.py
+
+ADD sign-metadata.sh /sign-metadata.sh
```

Comment:
Since we're here anyway, it'd probably be a good idea to switch to `COPY`.  The `ADD` command has some magic which does things like extract tar files, so the [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) is to not use it unless you need the magic.  The upstream files used ADD, but I think we've diverged enough that it's worth switching.

Either way, I also like the idea of just putting these two int something like `scripts` and doing a `COPY scripts/* /opt/pulp/bin`; the location of anything executable in the filesystem root makes my skin crawl - even if it is a container.  That's not so important on its own, but if (when) we end up needing these in the worker as well, they'll need moved to the pulp-core dockerfile so they're just in all of the containers.  Moving them into a directory under pulp-core and copying them to a location under /opt/pulp there means that can be done in the initial build image, where they'll be picked up in the second stage copy of /opt/pulp.